lib/arjdbc/abstract/core.rb

```diff
@@ -62,7 +62,7 @@ def translate_exception(exception, message:, sql:, binds:)
       end
 
       def extract_raw_bind_values(binds)
-        binds.map(&:value_for_database)
```

Comment:
Is this something only missing for postgresql?

Reply:
I'm really not sure. This change is not really relevant to the purpose of this PR.

Also, https://github.com/jruby/activerecord-jdbc-adapter/issues/1088#issuecomment-870094058 reports that this should not be an issue in Rails 6.1.4

Reply:
Ah yeah I guess we will not worry about that one.  I am trying to get our CI working again.  Looks like some changes in bundler and retrieving via git was messed up.  Once I get it ok I will rerun this.